styles.css

```diff
@@ -0,0 +1,340 @@
+    text-align: center;
+    justify-content: center;
+    max-width: 100%;
+    min-height: 350px;
+    color: #5722CD;
+}
+.section1-h2 {
+    padding: 15px;
+    max-width: 60%;
+    align-self: center;
+}
+.section1-p {
+    position: absolute;
+    top: 95%;
+    align-self: center;
+}
+/* --------------- 3.4 SECTION 2 ------------------- */
+.flex-container-section2 {
+    display: flex;
+    flex-direction: column;
+    background-color: #F3F6F9;
+}
+
+.outer-col-section2:nth-child(2) {
+    flex-direction: row-reverse;
+}
+.outer-col-section2 {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+}
+.inner-col-section2 {
+    flex-basis: 40%;
+    padding: 35px 20px;
+    align-self: center;
+    flex-grow: 0;
+    flex-shrink: 0;
+    height: 300px;
+    max-width: 250px;
+}
+.image-wrapper {
+    flex-basis: 50%;
+    flex-grow: 0;
+    flex-shrink: 1;
+    padding: 50px 30px;
+}
+.images-section2 {
+    width:100%;
+    box-shadow: 10px 10px 20px rgba(0, 0, 0, 0.5);
+}
+.link-project {
+    text-decoration: none;
+    color: #5722CD;
+    text-transform: uppercase;
+    font-size: 10px;
+    font-weight: bold;
+}
+.squiggle-section2 {
+    margin: 25px 0;
+}
+.section2-p {
+    margin-bottom: 20px;
+}
+/* --------------- 3.5 SECTION 3 ------------------- */
+/*krijg een onwijs grote gap tussen de bovenste rij en onderste rij blokjes, geen idee*/
+/*waar dat vandaan komt, daarom de container "ingeknepen", zodat je de gap niet ziet :)*/
+/*niet goed natuurlijk, daarnaast lukte het niet om de container niet te laten groeien/krimpen.*/
+/*Ik heb geprobeert om van de container een item te maken en vervolgens grow/shrink op nul, maar*/
+/*dat werkte niet. Zou wel moeten werken denk ik, weet niet wat ik fout deed en was te laat voor SME*/
+/*Enne nu je dit toch aan het lezen bent, heb het wel gezien in de opdracht maar het is zoals je*/
+/*kunt zien niet gelukt om weg te blijven van  letterlijke height: 500px properties*/
+.flex-container-section3 {
+    display: flex;
+    flex-direction: row;
+    min-height: 450px;
+    flex-wrap: wrap;
+    padding-top: 20px;
+    padding-bottom: 30px;
+    column-gap: 30px;
+    row-gap: 15px;
+    justify-content: center;
+}
+.outer-col-section3 {
+    display: flex;
+    flex-direction: column;
+    width: 200px;
+    height: 200px;
+    border: 1px solid #5722CD;
+    box-shadow: 5px 5px 10px rgba(0, 0, 0, 0.2);
+    flex-shrink: 0;
+    flex-grow: 0;
+    padding: 15px;
+}
+.img-section3 {
+    width: 50px;
+    height: 50px;
+}
+.outer-col-section3 h4{
+    color: #5722CD;
+    padding-bottom: 10px;
+    margin-top: 5px;
+}
+.outer-col-section3 p {
+    color: #7A7A7A;
+    font-size: 9px;
+}
+/* ---------------3.6 SECTION 3 ------------------- */
+form {
+    min-height: 450px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-evenly;
+
+    color: white;
+    font-family: inherit;
+    background-color: #0066ff;
```

Comment:
Het mooiste is om voor waarden die vaker voorkomen gebruik te maken van css-variabelen.
```
:root {
   --color-bg: #0066ff;
}

form {
   background-color: var(--color-bg);
}
```